ci-operator/config/openshift/cluster-api-provider-agent/openshift-cluster-api-provider-agent-release-ocm-2.8.yaml

```diff
@@ -0,0 +1,123 @@
+base_images:
+  assisted-image-service:
+    name: ocm-2.8
+    namespace: edge-infrastructure
+    tag: assisted-image-service
+  assisted-installer:
+    name: ocm-2.8
+    namespace: edge-infrastructure
+    tag: assisted-installer
+  assisted-installer-agent:
+    name: ocm-2.8
+    namespace: edge-infrastructure
+    tag: assisted-installer-agent
+  assisted-installer-controller:
+    name: ocm-2.8
+    namespace: edge-infrastructure
+    tag: assisted-installer-controller
+  assisted-service:
+    name: ocm-2.8
+    namespace: edge-infrastructure
+    tag: assisted-service
+  assisted-service-index:
+    name: assisted-service-index
+    namespace: edge-infrastructure
+    tag: ocm-2.8
+  assisted-service-scripts:
+    name: ocm-2.8
+    namespace: edge-infrastructure
+    tag: assisted-service-scripts
+  base:
+    name: "4.12"
+    namespace: ocp
+    tag: base
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  hypershift-operator:
+    name: "4.12"
+    namespace: ocp
+    tag: hypershift-operator
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  to: cluster-api-provider-agent
+promotion:
+  name: ocm-2.8
+  namespace: edge-infrastructure
+releases:
+  latest:
+    integration:
+      name: "4.12"
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: mirror-nightly-image
+  cron: '@daily'
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: cluster-api-provider-agent
+    env:
+      RELEASE_TAG_PREFIX: ocm-2.8
+    test:
+    - ref: assisted-baremetal-images-publish
+- as: mirror-vcsref-image
+  postsubmit: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: cluster-api-provider-agent
+    env:
+      RELEASE_TAG_PREFIX: ocm-2.8
+    test:
+    - ref: assisted-baremetal-images-publish
+- as: build
+  commands: |
+    export GOLANGCI_LINT_CACHE=/tmp/.cache
+    export GOPROXY=https://proxy.golang.org
+    HOME=/tmp make build
+  container:
+    from: src
+  skip_if_only_changed: (^(docs|config)/)|(\.md$)|((^|/)OWNERS$)
+- as: unit-test
+  commands: HOME=/tmp make test
+  container:
+    from: src
+  skip_if_only_changed: (^(docs|config)/)|(\.md$)|((^|/)OWNERS$)
+- as: e2e-ai-operator-ztp-capi
+  steps:
+    cluster_profile: packet-assisted
+    dependencies:
+      ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE: release:latest
+      HYPERSHIFT_IMAGE: hypershift-operator
+      INDEX_IMAGE: assisted-service-index
+      OPENSHIFT_INSTALL_RELEASE_IMAGE: release:latest
+      PROVIDER_IMAGE: pipeline:cluster-api-provider-agent
+    workflow: assisted-baremetal-operator-capi
+- as: e2e-ai-operator-ztp-capi-periodic
```

Comment:
if it runs periodically against assisted-service 2.8 I think we don't need this one

Reply:
will remove it from 2.7 as well